cmd/velero-plugin-for-vsphere/main.go

```diff
@@ -33,7 +33,8 @@ func main() {
 	veleroPluginServer = veleroPluginServer.
 		RegisterBackupItemAction("velero.io/vsphere-pvc-backupper", newPVCBackupItemAction).
 		RegisterRestoreItemAction("velero.io/vsphere-pvc-restorer", newPVCRestoreItemAction).
-		RegisterDeleteItemAction("velero.io/vsphere-pvc-deleter", newPVCDeleteItemAction)
+		RegisterDeleteItemAction("velero.io/vsphere-pvc-deleter", newPVCDeleteItemAction).
+		RegisterVolumeSnapshotter("velero.io/vsphere", newVolumeSnapshotterPlugin)
```

Comment:
Do we want to use the defined constant for the string of  "velero.io/vsphere"?

Reply:
When speaking to Nolan, I suggested we allow registering multiple versions of plugin, something like "velero.io/vsphere/1.1.0", so this is not yet a fixed name, I'll leave it as it is for now. He thought it was a good idea, and is probably looking into it.